cmd/harmony/main.go

```diff
@@ -72,7 +73,7 @@ var (
 	ip               = flag.String("ip", "127.0.0.1", "IP of the node")
 	port             = flag.String("port", "9000", "port of the node.")
 	logFolder        = flag.String("log_folder", "latest", "the folder collecting the logs of this execution")
-	freshDB          = flag.Bool("fresh_db", false, "true means the existing disk based db will be removed")
+	freshDB          = flag.Bool("fresh_db", true, "true means the existing disk based db will be removed")
```

Comment:
why change the default value?

Reply:
This is mistakenly committed code. I already cleaned it locally. [PR not ready for review yet]

Reply:
Test code, reverted.